packages/eslint-plugin/src/utils/constants.ts

```diff
@@ -1 +1,2 @@
 export const COMPILED_IMPORT = '@compiled/react';
+export const ATLASKIT_IMPORT = '@atlaskit/css';
```

Comment:
This looks odd. 
While `@atlaskit` is OSS too, I am not sure whether we should mix the two.

Why is this required?

Reply:
We already do in a few other places, and I even plan to remove the `importSources: ['@atlaskit/css']` config requirement in the 50+ configs we have to add it to internally and make that a defaulted alias 😄 so I approve…given Compiled is nearly exclusively used internally.

Alternatively, we'd want to configure things better internally, but this is easier.

Reply:
Plan actioned: https://github.com/atlassian-labs/compiled/pull/1720
(I will fix my PR or this PR, whichever one lands first)